docs/server/features/archiving.md

```diff
@@ -101,13 +113,16 @@ The panels are available in the `Events Served` section of the [miscellaneous pa
 
 This initial release has several limitations that we intend to improve in future releases.
 
-Work to improve the following limitations is about to begin or has already begun:
-- The headers of archived chunks are read on startup, just as for local chunks. This will increase startup times when there are a lot of chunks in the archive.
+Work to improve the following limitations is in progress:
+- The headers of archived chunks are read on startup, just as for local chunks. This will increase startup times significantly when there are a lot of chunks in the archive.
+- Requests that read the archive can cause other reads to be queued behind them, resulting in higher read latency if the archive is being accessed frequently.
+
+Work to improve the following limitations is planned:
 - Once uploaded to the archive, the chunks there are not scavenged any further.
 - Clients cannot yet opt out of their read reading from the archive.
 - Repeated reads of the same part of the archive are not cached locally.
-- Requests that read the archive can cause other reads to be queued.
 
-Work on the following items may be added according to interest
+Work on the following items may be added according to interest:
 
 - At the moment only S3 is supported. A local file-system based archive exists for development/testing purposes.
```

Comment:
Is the file-system based archive appropriate for on-premise setups where someone might want to archive their chunks to a local NAS ?